next.config.mjs

```diff
@@ -0,0 +1,35 @@
+import remarkGFM from 'remark-gfm';
+import remarkUnwrapImage from 'remark-unwrap-images';
+import createMDX from '@next/mdx';
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  pageExtensions: ['ts', 'tsx', 'mdx'],
+  // webpack: (config) => ({
+  //   ...config,
+  //   module: {
+  //     ...config.module,
+  //     rules: config.module.rules.concat([
+  //       {
+  //         test: /\.md/,
+  //         type: 'asset/source',
+  //       },
+  //     ]),
+  //   },
+  // }),
```

Comment:
Will this be uncommented at some point in the future?

Reply:
Nope, since MDX is supported by NextJS this won't be needed

_I think_